src/RoverExplorer/RoverExplorer.module.scss

```diff
@@ -1,3 +1,109 @@
-.roverExplorer {
-    background-color: blue;
+@import url('https://fonts.googleapis.com/css2?family=Rajdhani:wght@700&display=swap');
+@import "../generalStyle.scss";
+
+@mixin GridItem {
+    align-items: center;
+    text-align: center;
+    vertical-align: middle;
+    line-height: 200px;
+    font-size: 20px;
+    color: $colorTextLight;
+    border-radius: 3%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    border-color: white;
+    border-style: inset;
+    border-width: 0.1px;
+    height: 100%;
+    @include subtitle;
+}
+
+
+
+.Content {
+    background: url("https://s3.amazonaws.com/psiu/wallpapers/ESP_018820_2035/ESP_018820_2035_phone.jpg");
```

Comment:
sorry - missed this before.
Could we move these images into our own repo?

ie could we:
- save the images
- pop them in the `public/images` folder in our own repo
- then reference that URL rather than the random one on the internet.

Just because that means the images are under our control and can't change to something we don't want / dissapear at some point in the future